daal4py/sklearn/cluster/_dbscan.py

```diff
@@ -29,6 +29,8 @@
 from .._device_offload import support_usm_ndarray
 from .._utils import sklearn_check_version
 
+import numbers
+
```

Comment:
```suggestion
```

Reply:
There is one function "def" right after the import section (not shown in the above view). Line 33 is one of the two blank lines required before "def" by pep8.